lib/src/package.dart

```diff
@@ -109,7 +110,8 @@ class Package {
   List<String> get executablePaths {
     final binDir = p.join(dir, 'bin');
     if (!dirExists(binDir)) return <String>[];
-    return ordered(listDir(p.join(dir, 'bin'), includeDirs: false))
+    return listDir(p.join(dir, 'bin'), includeDirs: false)
+        .sorted()
         .where((executable) => p.extension(executable) == '.dart')
         .map((executable) => p.relative(executable, from: dir))
```

Comment:
Can sort after `where`  here (and `map`, unless it matters whether it's sorted as absolute or relative path, which I hope it shouldn't). Then there should be fewer elements to sort.

And one less `toList` to do:
```dart
  listDir(...).where(...).map(...).toList()..sort()
```
or 
```dart
    return [for (var executable in listDir(...))
      if (p.extension(executable) == '.dart')
        p.relative(executable, from: dir),
    ]..sort();
```